src/runtime/entitlements-manager-test.js

```diff
@@ -86,6 +86,10 @@ describes.realWin('EntitlementsManager', {}, (env) => {
     eventManager = new ClientEventManager(Promise.resolve());
     eventManagerMock = sandbox.mock(eventManager);
 
+    sandbox.stub(win, 'setTimeout').callsFake((callback) => {
```

Comment:
LGTM as a temp fix - drop a comment and/or todo to clean up later and implement a general solution to flush timers perhaps?

Reply:
Sounds good :+1: